examples/pom.xml

```diff
@@ -77,23 +77,10 @@
 					</execution>
 				</executions>
 			</plugin>
-			<plugin>
```

Comment:
Let's keep this plugin, I want to enforce code style.

Reply:
Brendan-san, I understood about it !!